tests/functional/baseline-no-breadcrumbs/collections/ns2/col/changelog.rst

```diff
@@ -0,0 +1,4 @@
+
+The changelog of ns2.col could not be rendered::
+
+  list index out of range
```

Comment:
is this intentional? part of a test case?

Reply:
This is intentional, for this collection the changelog is broken (changelogs/changelog.yaml contains invalid data which triggers a crash).